docs/user/checks.rst

```diff
@@ -62,3 +62,50 @@ parameters used for iperf3 checks (e.g. timing, port, username, password,
     may need to update the :doc:`metric configuration
     <device-checks-and-alert-settings>` to enable alerts for the iperf3
     check.
+
+.. _wifi_client_check:
+
+WiFi Client
+-----------
+
+This check sends alerts based on the total number of WiFi Clients
+connected to a device. It sends two types of alerts:
+
+- **Maximum WiFi Users**: When the total number of WiFi clients connected
```

Comment:
```suggestion
- **Maximum WiFi Clients**: When the total number of WiFi clients connected
```

I recommend consistency with the wording to avoid confusing readers.